Sources/Contour/Configuration/IBusConfigurator.cs

```diff
@@ -16,11 +16,6 @@ namespace Contour.Configuration
     /// </summary>
     public interface IBusConfigurator
     {
-        /// <summary>
-        /// The enable caching.
-        /// </summary>
-        void EnableCaching();
```

Comment:
Breaking backward compatibility. You should avoid it.

Reply:
Reverted deletion and marked as obsolete.